### PR TITLE
improvements from linters pylint and flake8

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

File: tests/test_bootstrap4/test_bootstrap.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def test_deprecate_bootstrap_class(self):
 
         app = Flask(__name__)
         with pytest.warns(UserWarning):
-            bootstrap = Bootstrap(app)  # noqa: F841
+            bootstrap = Bootstrap(app)  # noqa: F841  # pylint:disable=unused-variable
```
Comment (greyli) [inline tests/test_bootstrap4/test_bootstrap.py:14]: This project doesn't use Pylint, the pylint comments are useless. Please remove them.

File: tests/test_bootstrap4/test_render_table.py [hunk 2]
```diff
@@ -182,12 +182,10 @@ def test():
         db.drop_all()
         db.create_all()
         for i in range(10):
-            m = Message(
-                text=f'Test message {i+1}',
-                sender='me',
-                recipient='john_doe'
-            )
-            db.session.add(m)
+            msg = Message(text=f'Test message {i+1}',
```
Comment (greyli) [inline tests/test_bootstrap4/test_render_table.py:185]: Keep the original indentation.

**Comment by greyli** [comment]
You could use `fixes #XXX` in the description when creating the PR (replace the `XXX` with the issue number), this will link the related issue and close the linked issue when the PR is merged.

**Review by greyli — CHANGES_REQUESTED** [review]

File: tests/test_bootstrap4/test_bootstrap.py (hunk 1 above)
Comment (PanderMusubi) [reply tests/test_bootstrap4/test_bootstrap.py]: Done

File: tests/test_bootstrap4/test_render_table.py (hunk 2 above)
Comment (PanderMusubi) [reply tests/test_bootstrap4/test_render_table.py]: Done

**Comment by PanderMusubi** [comment]
Reworked and let me know if you want the commits squashed to one.

**Comment by greyli** [comment]
It's OK, I can use the squash merge option. Thanks!

**Review by greyli — APPROVED** [review]

**Pull request merged**